config/routes.rb

```diff
@@ -1,21 +1,24 @@
 Rails.application.routes.draw do
-
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
   root "welcome#index"
 
-  # resources :dashboard, as: "merchant_dashboard"
-  resources :merchants do
-    member { get "dashboard"}
-    resources :items, controller: "merchant_items", only: [:index]
-    resources :invoices, controller: "merchant_invoices", only: [:index, :show]
+  resources(
+    :merchant,
+    controller: "merchant_site/merchant",
+    only: %i[index show]
+  ) do
```

Comment:
lines 5-9 is for `/merchant`, `/merchant/:id`, etc. It uses the `merchant_site/merchant_controller.rb`